test/testdrive/source-linear-operators.td

```diff
@@ -225,25 +225,23 @@ Source materialize.public.data (UID):
 | Project (#0..=#2)
 
 Query:
-%0 = Let l0 =
+%0 =
```

Comment:
This change is due to our lack of proper support for CSE when it comes to MapFilterProjects with some parts in common, so changing the `Filter-Map-Filter-Project` sequence to `Map-Filter-Project` causes CSE to stop kicking in for some common filters but to start kicking in for some common maps. 

The ticket to properly handle CSE in MapFilterProjects is https://github.com/MaterializeInc/materialize/issues/7825.